--- xf86-video-rendition-4.2.5/src/rendition.h~	2012-07-16 02:42:28.000000000 +0000
+++ xf86-video-rendition-4.2.5/src/rendition.h	2013-03-24 14:08:26.645893429 +0000
@@ -28,9 +28,6 @@
 /* All drivers initialising the SW cursor need this */
 #include "mipointer.h"
 
-/* All drivers implementing backing store need this */
-#include "mibstore.h"
-
 /* All drivers using the mi colormap manipulation need this */
 #include "micmap.h"
 
--- xf86-video-rendition-4.2.5/src/rendition.c~	2012-07-17 07:32:12.000000000 +0200
+++ xf86-video-rendition-4.2.5/src/rendition.c	2013-04-06 02:59:55.190719126 +0200
@@ -1173,7 +1173,6 @@ renditionScreenInit(SCREEN_INIT_ARGS_DEC
     fbPictureInit (pScreen, 0, 0);
 
     xf86SetBlackWhitePixels(pScreen);
-    miInitializeBackingStore(pScreen);
    
     /*********************************************************/
     /* The actual setup of the driver-specific code          */
